src/ssl.c

```diff
@@ -13959,6 +13959,9 @@ int wolfSSL_DTLS_SetCookieSecret(WOLFSSL* ssl,
 
         case ACCEPT_FIRST_REPLY_DONE :
             if ( (ssl->error = SendServerHello(ssl)) != 0) {
+            #ifdef WOLFSSL_CHECK_ALERT_ON_ERR
+                ProcessReplyEx(ssl, 1); /* See if an alert was sent. */
```

Comment:
When was the alert read in?

Reply:
Reads are typically blocking.
If the connection isn't closed it could block indefinitely.

Reply:
Given that you need to define WOLFSSL_CHECK_ALERT_ON_ERR, then we will let the customer figure out how to handle this.